clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -779,6 +779,9 @@ static void InitializeCPlusPlusFeatureTestMacros(const LangOptions &LangOpts,
 
   // TODO: Final number?
   Builder.defineMacro("__cpp_type_aware_allocators", "202500L");
+
+  if (LangOpts.ImplicitConstexpr) // same value as GCC
+    Builder.defineMacro("__cpp_implicit_constexpr", "20211111");
```

Comment:
We don't usually define macros for non-standard feature. Instead users should use `__has_extension(cxx_implicit_constexpr)`

Reply:
this is to mirror GCC's behavior, but I'm happy to remove it